com.unity.ml-agents/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to
 - `max_step` in the `TerminalStep` and `TerminalSteps` objects was renamed `interrupted`.
 - `beta` and `epsilon` in `PPO` are no longer decayed by default but follow the same schedule as learning rate. (#3940)
 - `get_behavior_names()` and `get_behavior_spec()` on UnityEnvironment were replaced by the `behavior_specs` property. (#3946)
+- `use_visual` and `allow_multiple_visual_obs` are replaced by `allow_multiple_obs` which allows
```

Comment:
nit: clarify that this is for the gym wrapper (maybe we should split this section up further).